.gitignore

```diff
@@ -1,6 +1,7 @@
 .DS_Store
 node_modules
 /dist
+.prettierrc
```

Comment:
This should not be part of this PR / branch.
For now okay. But please don't do this in the future.

A better way: commit on dev or a different branch and merge.